STAR.Web/Content/Site.css

```diff
@@ -3,27 +3,96 @@
     padding-bottom: 20px;
 }
 
+/* This isn't necessary, bootstrap takes care of this.*/
 /* Set padding to keep content from hitting the edges */
-.body-content {
+/*.body-content {
```

Comment:
It's OK to just remove this code instead of commenting.  You can place a comment on it in github (like this one) if you feel you need to communicate the reason why.